rotkehlchen/api/rest.py

```diff
@@ -4106,6 +4106,26 @@ def get_manual_latest_prices(
                 'to_asset': price_entry[1],
                 'price': price_entry[2],
             })
+
+        if self.rotkehlchen.chains_aggregator.get_module('nfts') is not None:
+            # query also nfts manual prices
+            nft_price_data = self._eth_module_query(
```

Comment:
Does this also query the nft prices? Or just read from the DB? From a quick look it just reads from the DB, which should be okay.

Also why go through `_eth_module_query()` and not query `get_nfts_with_price` directly? What's the reasoning of the extra work? You are actually reading just the `result` of the dict and not using the constructured api response at all.

Reply:
It queries from the database. Okey